.build-emscripten.sh

```diff
@@ -36,8 +36,17 @@ pushd src/h3lib/lib
 cp ../../../../build/sizes.h ../include
 cp ../../../../build/sizes.c .
 # Compile with emscripten
-emcc -O3 -I ../include *.c -o libh3.js -DH3_HAVE_VLA -s INVOKE_RUN=0 -s EXPORT_NAME="'libh3'" -s MODULARIZE=1 -s NO_FILESYSTEM=1 -s NODEJS_CATCH_EXIT=0 -s TOTAL_MEMORY=33554432 -s ALLOW_MEMORY_GROWTH=1 -s WARN_UNALIGNED=1 -s EXPORTED_FUNCTIONS=$bound_functions -s EXTRA_EXPORTED_RUNTIME_METHODS='["cwrap", "getValue", "setValue"]' --memory-init-file 0
+emcc -O3 -I ../include *.c -o libh3.js -DH3_HAVE_VLA -s WASM=0 -s INVOKE_RUN=0 -s EXPORT_NAME="'libh3'" -s MODULARIZE=1 -s NO_FILESYSTEM=1 -s NODEJS_CATCH_EXIT=0 -s TOTAL_MEMORY=33554432 -s ALLOW_MEMORY_GROWTH=1 -s WARN_UNALIGNED=1 -s EXPORTED_FUNCTIONS=$bound_functions -s EXTRA_EXPORTED_RUNTIME_METHODS='["cwrap", "getValue", "setValue"]' --memory-init-file 0
 cp libh3.js ../../../../out/libh3.js
-echo "module.exports = libh3();" >> ../../../../out/libh3.js
+cat << EOF >> ../../../../out/libh3.js
+const h3 = libh3();
+module.exports = h3;
+// Regression in latest Emscripten drops these methods, re-attach to the object with proper names
```

Comment:
Something in the build objects to this comment - maybe just use a `#` comment in the script instead?

Reply:
Very strange, it built and ran tests locally just fine for me. Let me investigate a bit more, I'd prefer to have that comment in the output file as well for that extra level of documentation.

Reply:
Ahh, I see. I didn't commit the `out/libh3.js` file in that last commit and it aborted because of the difference between the two, should be quick to fix.

Reply:
Yes, sorry, just realized that as well. That should fix the build.